requirements.txt

```diff
@@ -1,21 +1,44 @@
+# Melissa Core Dependencies
+wheel==0.24.0
+six==1.10.0
+oauthlib==1.0.3
+requests-oauthlib==0.5.0
+Wave==0.0.2
+PyAudio==0.2.9
+pygame==1.9.2b6
```

Comment:
I shall do that and start a new pull request. Is there a way to avoid another pull request?